tests/subsys/settings/fcb/raw/prj_native_posix.conf

```diff
@@ -1,5 +1,4 @@
 CONFIG_ZTEST=y
-CONFIG_STDOUT_CONSOLE=y
```

Comment:
Question, did you investigate whether [testcase.yaml](https://github.com/zephyrproject-rtos/zephyr/blob/3631fc4ed87d93cceb4affeb2843ff91897ce430/tests/subsys/settings/fcb/raw/testcase.yaml) should be updated and split in two parts.

One for the native posix platforms, and one for other platforms where you can then add:
```
extra_configs:
- CONFIG_STDOUT_CONSOLE=y
```

I assume `CONFIG_STDOUT_CONSOLE=y` was added originally for a reason.

Reply:
I did not, seemed unrelated. I went ahead and added that as a separate commit on this PR to clean it up.